environments/operator/obs-operator.jsonnet

```diff
@@ -1,3 +1,18 @@
-(import '../../components/observatorium.libsonnet') + {
-  config+:: (import 'operator-config.libsonnet'),
-} + (import '../../components/observatorium-configure.libsonnet')
+local config = import 'operator-config.libsonnet';
+local manifests = ((import '../../components/observatorium.libsonnet') + {
+                     config+:: config,
+                   } + (import '../../components/observatorium-configure.libsonnet')).manifests;
+
+local addOwner(k, v) = v { metadata+: { ownerReferences: [{
+  apiVersion: config.apiVersion,
+  blockOwnerdeletion: true,
+  controller: true,
+  kind: config.kind,
+  name: config.name,
+  uid: config.uid,
+}] } };
+local ownermanifests = std.mapWithKey(addOwner, manifests);
```

Comment:
as you're not using the key, you can just use `std.map`

Reply:
std.map works for array.
This is an object here:
`RUNTIME ERROR: std.map second param must be array / string, got object`

Reply:
ah ok yes, makes sense :+1: